src/commands/bundle.ts

```diff
@@ -77,10 +77,9 @@ export default class Bundle extends Command {
       this.log(`Check out your shiny new bundled files at ${output}`);
     }
 
-    const result = await load(output);
-
     // Metrics recording.
-    await this.recordActionExecuted('bundle', {success: true, files: AsyncAPIFiles.length}, result.text());
+    this.specFile = await load(output);
+    this.metricsMetadata = {success: true, files: AsyncAPIFiles.length};
```

Comment:
Yout don't need the `success` field in any command since https://github.com/smoya/cli/pull/8/files#diff-440b55a9ff9e0486020fc49197f3278d14446306fc0c5d5107b48139c20bfda6R77